core/js/pokemon.maps.js

```diff
@@ -420,3 +442,29 @@ function isMobileDevice() {
     //  Basic mobile OS (not browser) detection
     return (/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent))
 }
+
+function coordinate_area(){
+	$.ajax({
+		'async': false,
```

Comment:
We shouldn't do anything with `'async': false`. It's deprecated and slow. Please move this to initMap() and make it async.